sample_app/sampleapp_common.c

```diff
@@ -717,13 +717,19 @@ static int app_state_ind (
                switch (err_code)
                {
                case PNET_ERROR_CODE_2_ABORT_AR_CONSUMER_DHT_EXPIRED:
-                  error_description = "AR_CONSUMER_DHT_EXPIRED";
+                  error_description = "ABORT_AR_CONSUMER_DHT_EXPIRED";
```

Comment:
"PLC terminated the connection due to missed cyclic data deadline"

Reply:
done, I have updated all descriptions